Gaijinizer patch/Map530.txt

```diff
@@ -8,709 +8,908 @@
 
 > BEGIN STRING
 か、帰ってこれ…た…？
-> CONTEXT: Map530/events/7/pages/0/10/Dialogue/ruka_fc1 < UNTRANSLATED
-
+> CONTEXT: Map530/events/7/pages/0/10/Dialogue/ruka_fc1
+A-Are we... back?#Ron
 > END STRING
 
 > BEGIN STRING
 一応、アバドーンの外には出たって感じかな……
 
 まだ多元宇宙の空間だけど……
-> CONTEXT: Map530/events/7/pages/0/12/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map530/events/7/pages/0/12/Dialogue/enemy29
+It feels like we're out of Abaddon now at the
+very least. We're still in the multiverse space,
+though.#Ron
 > END STRING
 
 > BEGIN STRING
 あー、でも、久々にすがすがしい気分♪
 
 やっぱり五体満足って快適だわー
 三つ編みで動くのはもうコリゴリだよ。
-> CONTEXT: Map530/events/7/pages/0/16/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map530/events/7/pages/0/16/Dialogue/enemy29
+Aah, but still, I haven't felt this refreshed
+in a long time. It really does feel great to be
+in peak physical form. I was getting sick of
+having to use my braids to move. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 はぁ……
 結局、一緒に連れてこられちゃったわね……
 
 萎えちゃったから帰るわ
-> CONTEXT: Map530/events/7/pages/0/25/Dialogue/enemy2 < UNTRANSLATED
+> CONTEXT: Map530/events/7/pages/0/25/Dialogue/enemy2
+Haah... So you ended up bringing me with you...
 
+I'm going home now that I've lost all my
+motivation.#Ron
 > END STRING
 
 > BEGIN STRING
 アバドーンをぶっ潰すの、後はあんたらに任せるわよ。
 
 今度はしくじらないようにね。
-> CONTEXT: Map530/events/7/pages/0/30/Dialogue/enemy2 < UNTRANSLATED
+> CONTEXT: Map530/events/7/pages/0/30/Dialogue/enemy2
+I'll leave the Abaddon beat-down to you guys.
 
+Try not to mess up this time.#Ron
 > END STRING
 
 > BEGIN STRING
 ヘカーテ……一つだけ教えてくれないか？
 
 キミは……僕に復讐をするときに
 なぜ\\C[24]あの力\\C[0]で僕を襲わなかったんだ？
-> CONTEXT: Map530/events/7/pages/0/44/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map530/events/7/pages/0/44/Dialogue/kazuya
+Can I ask you something before you go?
 
+Why didn't you use \\C[24]that power \\C[0]when you were
+trying to get your revenge on me?#Ron
 > END STRING
 
 > BEGIN STRING
 \\C[2]あんな力があるなら、わざわざオールドワンに加担せずとも
 
 僕を始末することぐらい、簡単だったはずじゃ……
-> CONTEXT: Map530/events/7/pages/0/49/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map530/events/7/pages/0/49/Dialogue/kazuya
+\\C[2]You wouldn't have had to turn to the Old One for
+help if you'd used it, and dealing with me
+would've been a cinch.#Ron
 > END STRING
```

Comment:
Slight rewording for flow:
```
\\C[2]You wouldn't have had to turn to the Old One for
help if you'd used it. 

It would have been easy to deal with me.
```